stream.go

```diff
@@ -200,6 +201,10 @@ START:
 	// Send the header
 	s.sendHdr.encode(typeData, flags, s.id, max)
 	if err = s.session.waitForSendErr(s.sendHdr, body, s.sendErr); err != nil {
+		if errors.Is(err, ErrSessionShutdown) || errors.Is(err, ErrConnectionWriteTimeout) {
```

Comment:
This makes yamux depend on Go 1.13 or later. That's not necessarily a problem, but it should be noted as such.

Reply:
I don't even think it needs to be noted, TBH. Go 1.13 is _well_ outside the security update window and should not be used any longer (or for the past year +)